packages/compass-schema/src/components/minichart/minichart.jsx

```diff
@@ -34,13 +34,13 @@ class MiniChart extends PureComponent {
     // but it is not noticable to the user.
     this.resizeListener();
     window.addEventListener('resize', this.resizeListener);
-    this.unsubscribeMiniChartResize =
-      this.props.actions.resizeMiniCharts.listen(this.resizeListener);
+    // this.unsubscribeMiniChartResize =
```

Comment:
From some old comment in the code:

```
   * Called when the window size changes or via the resizeMiniCharts action,
   * triggered by index.jsx. Only redraw if the size is > 0.
```

I think we removed this explicit action calling at some point after refactoring the tabs to properly unmount, but didn't cleanup the action itself, should be safe to completely drop